modules/URL/ConferenceUrl.js

```diff
@@ -0,0 +1,73 @@
+
+import { redirect } from '../util/helpers';
+
+/**
+ * Stores the original conference room URL with all parameters.
+ * @type {string}
+ */
+let originalURL;
+
+/**
+ * A simplified version of the conference URL stripped out of the parameters
+ * which should be used for sending invites.
+ * @type {string}
+ */
+let inviteURL;
+
+/**
+ * The modules stores information about the URL used to start the conference and
+ * provides utility methods for dealing with conference URL and reloads.
+ */
+export default {
```

Comment:
I think it's better to define a class ConferenceUrl and create a new instance of ConferenceUrl in app.js file. Also I would assign the instance to the global APP variable, this way we can save few imports. 

Most of the modules are assigned to and accessible from the global APP variable. In order to keep our code consistent I think it's going to be good to apply this for the URL module too.